apps/public-docsite-v9/src/Concepts/Upgrade/FromV0/Components/Checkbox.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Upgrading/from v0/Components/Checkbox Upgrade" />
+
+# Checkbox Upgrade
+
+Fluent UI Northstar (v0) provides the `Checkbox` control to make a choice between two mutually exclusive options. Fluent UI v9 provides a `Checkbox` control, but has a different API.
+
+The main difference between v0 and v9 `Checkbox` is that v9 offers a circular variant and does not provide a toggle prop.
+
+## Examples
+
+### Basic Upgrade
+
+Basic usage of `Checkbox` v0
+
+```tsx
+import React from 'react';
+import { Checkbox } from '@fluentui/react-northstar';
+
+const CheckboxV0BasicExample = () => {
+  return (
+    <>
+      <Checkbox checked label="Checked" />
+      <Checkbox disabled label="Disabled" />
+      <Checkbox checked="mixed" label="Mixed" />
+    </>
+  );
+};
+```
+
+An equivalent `Checkbox` in v9 is
+
+```tsx
+import * as React from 'react';
+import { Checkbox } from '@fluentui/react-components/unstable';
+
+const CheckboxV9BasicExample = () => {
+  return (
+    <>
+      <Checkbox checked label="Checked" />
+      <Checkbox disabled label="Disabled" />
+      <Checkbox checked="mixed" label="Mixed" />
+    </>
+  );
+};
+```
+
+## Prop Mapping
+
+This table maps `Checkbox` v0 props to the `Checkbox` v9 equivalent.
+
+| v0               | v9               | Notes                                                       |
+| ---------------- | ---------------- | ----------------------------------------------------------- |
+| `accessibility`  | n/a              |                                                             |
+| `as`             | n/a              |                                                             |
+| `checked`        | `checked`        | Mutually exclusive with `defaultChecked`                    |
+| `className`      | `className`      |                                                             |
+| `defaultChecked` | `defaultChecked` | Mutually exclusive with `checked`                           |
+| `design`         | n/a              |                                                             |
+| `disabled`       | `disabled`       |                                                             |
+| `indicator`      | `indicator`      |                                                             |
+| `key`            | `key`            |                                                             |
+| `label`          | `label`          |                                                             |
+| `labelPosition`  | `labelPosition`  |                                                             |
+| `onChange`       | `onChange`       |                                                             |
+| `onClick`        | `key`            | v9 uses the intrinsic React prop                            |
```

Comment:
I think the v9 column should be `onClick` and the "notes" column should be for the `key` row above.